src/test/java/io/appium/java_client/android/FingerPrintTest.java

```diff
@@ -62,77 +65,76 @@
         }
     }
 
+    private AndroidElement findElementByText(String text) {
+        return driver.findElements(id("android:id/title")).stream().filter(androidElement ->
+                text.equals(androidElement.getText())).findFirst()
+                .orElseThrow(() ->
+                        new NoSuchElementException(String.format("There is no element with the text '%s'", text)));
+    }
```

Comment:
I was so used to localization testing that my mind automatically blocked using text to find an element :)
using text is much better and easier in this case 👍